### ISSUE TITLE
Do different pre-trained models have a big impact

### ISSUE THREAD

**Issue by cs8105** [bnu-wangxun/Deep_metric#38]
I tried to replace the BN-inception with resnet50 pretrained on imagenet. 
it seems that the performence droped a lot.
With your code, I can reproduce the result on cub-200, about 0.65.
And I inplemented a gluon(mxnet) version of the binomial loss.
But i can't find the pretrained model on mxnet.
Therefor, I tried different pretrained models. And the performences varies from 0.50 - 0.60 on cub-200.
Thus, I wonder know if different models have a big impact on the permformece?
Thank you!

**Comment by bnu-wangxun** [comment]
Different network need different hyperparamter: we also use resnet-50 and the performance is comparable with BN-inception on cars-196.

**Comment by cs8105** [comment]
Maybe I have found the reason.
the BN-inception you used is different from what I [used.](https://github.com/osmr/imgclsmob/blob/master/gluon/gluoncv2/models/bninception.py)
params of your BN-inception is about 22M.(pytorch)
while I used is about 11M. best performence: 60%.(gluon)
I just want to implement it on gluon(mxnet). But I cannot find a right pretrained model.
Thank you!

**Comment by ZhangHZ9** [comment]
Cub-200 is very small and the results may be affected by many details. I recommend you compare the performance with and without XBM under the same conditions except for the learning rate. Experiments on SOP or In-shop will be more stable.

**Comment by fengshi-cherish** [comment]
Is there any suggestion about the  hyperparamters in training resnet50. When i used ms-loss with resnet50 in CUB-200, the performance was dropped a lot, about 0.52 in rank1.I'm not sure what make such difference, hyperparameters or my code?
